python/BUSINESS_SUPPORT_OR_ABOVE_ENABLED/business-support-enabled.py

```diff
@@ -0,0 +1,415 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+'''
+  #####################################
+  ##           Gherkin               ##
+  #####################################  
+  Rule Name:
+  BUSINESS_SUPPORT_OR_ABOVE_ENABLED
+  
+  Description:
+  Check whether the AWS Account has Business or above(Enterprise) support plan  is enabled

```

Comment:
Check whether the AWS Account is subscribed to the AWS Business Support Plan or above (i.e. Enterprise).

Reply:
changed as per suggestion